### PR TITLE
Feature/external data

### PR DESCRIPTION
**Pull request by kfwilson**
this adds a bunch of external data to the repo - you can pretty much ignore the csvs. The important part is the cli + utilities code I wrote to transform and format the data and patch it into artifacts.

### CODE REVIEW

File: src/vivarium_dcpn_hypertension_mgt/external_data/drug_efficacy.csv [hunk 1]
```diff
@@ -0,0 +1,6 @@
+name,half_dose_efficacy_mean_mean,half_dose_efficacy_mean_lb,half_dose_efficacy_mean_ub,half_dose_efficacy_sd_mean,half_dose_efficacy_sd_sd,standard_dose_efficacy_mean_mean,standard_dose_efficacy_mean_lb,standard_dose_efficacy_mean_ub,standard_dose_efficacy_sd_mean,standard_dose_efficacy_sd_sd,double_dose_efficacy_mean_mean,double_dose_efficacy_mean_lb,double_dose_efficacy_mean_ub,double_dose_efficacy_sd_mean,double_dose_efficacy_sd_sd,half_dose_side_effects_mean,half_dose_side_effects_lb,half_dose_side_effects_ub,standard_dose_side_effects_mean,standard_dose_side_effects_lb,standard_dose_side_effects_ub,double_dose_side_effects_mean,double_dose_side_effects_lb,double_dose_side_effects_ub
+"thiazide_type_diuretics (Diuretics, DU)",7.4,6.6,8.2,0.281,0,8.8,8.3,9.4,0.281,0,10.3,9.4,11.2,0.281,0,2,-2.2,6.3,9.9,6.6,13.2,17.8,11.5,24.2
```
Comment (halfhorst) [inline src/vivarium_dcpn_hypertension_mgt/external_data/drug_efficacy.csv]: are the extra quotes here gonna screw you up ?

Comment (kfwilson) [reply src/vivarium_dcpn_hypertension_mgt/external_data/drug_efficacy.csv]: Nope good catch though. I don't really care about that column anyways. I have separate columns that explicitly code the drug combo - that one is just a human readable one that Meredith put in.

File: src/vivarium_dcpn_hypertension_mgt/tools/utilities.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    for category in non_guideline_profiles.hypertension_drug_category.unique():
+        category_profiles = non_guideline_profiles[non_guideline_profiles.hypertension_drug_category == category]
+
+        collapsed_profile = category_profiles.iloc[[0]]
+        collapsed_profile['drug_class'] = 'other'
+        collapsed_profile['percentage_among_therapy_category_mean'] = \
+            category_profiles.percentage_among_therapy_category_mean.sum()
+        collapsed_profile['percentage_among_therapy_category_sd'] = \
+            (category_profiles.percentage_among_therapy_category_sd ** 2).sum()
```
Comment (halfhorst) [inline src/vivarium_dcpn_hypertension_mgt/tools/utilities.py]: I think you need to root this again to get back to sd. You have the variance

Comment (kfwilson) [reply src/vivarium_dcpn_hypertension_mgt/tools/utilities.py]: yep definitely true. thanks for catching that!

**Pull request merged**